pkg/ddevapp/global_dotddev_assets/commands/host/phpmyadmin

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Prompt downloading phpMyAdmin if it is not installed.
+## Usage: phpmyadmin
+## Example: "ddev phpmyadmin"
+
+if [ "${DDEV_NONINTERACTIVE:-}" != "" ]; then
+  echo "Nothing has been changed."
+  exit 0
+fi
+
+read -p "phpMyAdmin is not installed would you like to install? (Y/n): [Y]" answer
+if [ -z "$answer" ]; then
+    answer="Y"
+fi
```

Comment:
Sorry for the noise, but:

There was no need to change this prompt, `no` here is a reasonable default in case someone accidentally runs this command and hits enter.

And the next prompt for restart should be `yes` by default, which is absolutely expected to run after `ddev get`.

Reply:
I had actually asked @rfay his opinion in slack. I think since you're trying to run the command it's reasonable for the default to be yes for this one too. I'll change it back if there is consensus.

Reply:
On second thought, I agree. It's unlikely that someone will type `ddev phpmyadmin` and press enter without knowing it.